src/nvidia-install.sh

```
@@ -0,0 +1,2649 @@
+                exit 0
+                ;;
+            --rollback)
+                rollback_installation
+                exit 0
+                ;;
+            --show-exit-codes)
+                show_exit_codes
+                exit 0
+                ;;
+            -n|--dry-run)
+                DRY_RUN="echo"
```

Shellcheck (0.11.0)

[warning] DRY_RUN appears unused. Verify use (or export if used externally).

(SC2034)